libraries/abstractions/pkcs11/test/iot_test_pkcs11.c

```diff
@@ -1834,8 +1834,10 @@ TEST( Full_PKCS11_EC, AFQP_GenerateKeyPair )
     /* mbedTLS structures for verification. */
     uint8_t ucSecp256r1Oid[] = pkcs11DER_ENCODED_OID_P256; /*"\x06\x08" MBEDTLS_OID_EC_GRP_SECP256R1; */
 
-    xResult = prvDestroyTestCredentials();
-    TEST_ASSERT_EQUAL_MESSAGE( CKR_OK, xResult, "Failed to destroy credentials before Generating Key Pair" );
+    /* TODO: We should be destroying credentials first, just in case. Currently a port cannot do this, but once that is complete this will need to be updated.
```

Comment:
Why is this important?  Is `xProvisionGenerateKeyPairEC()` not guaranteed to work?  If it can fail and this can only be recovered by destroying credentials, and the pkcs11 abstraction does not have this operation – please create a "feature request" issue to add it in this repo with a link to this PR, and include a link to the feature request in this `TODO` comment.  This will help us ensure that this gets `TODONE`. :-)

If `prvDestroyTestCredentials()` cannot fail in a complete/working port, then let's delete lines 1837-1840 rather than leave a `TODO` in here for something we're not certain is actually a problem.